### PR TITLE
PMM-11219: telemetry-ia-fix

### PR DESCRIPTION
**Pull request by ritbl**
[PMM-11219](https://jira.percona.com/browse/PMM-11219)

Build:

- [ ] https://github.com/Percona-Lab/pmm-submodules/pull/2946

### CODE REVIEW

File: managed/services/telemetry/config.default.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ telemetry:
   # PMM - generic information
   - id: PMMServerIAEnabled
     source: PMMDB_SELECT
-    query: (CASE WHEN alerting->'disabled' = 'false' THEN '1' ELSE '0' END) AS ia_enabled FROM settings s, jsonb_extract_path(s.settings, 'alerting') AS ia
+    query: (CASE WHEN ia -> 'enabled' = 'true' THEN '1' ELSE '0' END) AS ia_enabled FROM settings s, jsonb_extract_path(s.settings, 'ia') AS ia
```
Comment (artemgavrilov) [inline managed/services/telemetry/config.default.yml:5]: `ia` in settings was replaced with `alerting`. This change was added in 2.32 as I remember.

**Review by artemgavrilov — CHANGES_REQUESTED** [review]

File: managed/services/telemetry/config.default.yml (hunk 1 above)
Comment (ritbl) [reply managed/services/telemetry/config.default.yml]: for some reason we have old grafana in dev-container (grafana 8.3.10), while it should be 9.2.5.

File: managed/services/telemetry/config.default.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ telemetry:
   # PMM - generic information
   - id: PMMServerIAEnabled
     source: PMMDB_SELECT
-    query: (CASE WHEN alerting->'disabled' = 'false' THEN '1' ELSE '0' END) AS ia_enabled FROM settings s, jsonb_extract_path(s.settings, 'alerting') AS ia
+    query: (CASE WHEN ia->'disabled' = 'false' THEN '1' ELSE '0' END) AS ia_enabled FROM settings s, jsonb_extract_path(s.settings, 'alerting') AS ia
```
Comment (artemgavrilov) [inline managed/services/telemetry/config.default.yml:5]: I suggest to eliminate `ia` abbreviation, there is no more such system.
```suggestion
    query: (CASE WHEN alerting->'disabled' = 'false' THEN '1' ELSE '0' END) AS ia_enabled FROM settings s, jsonb_extract_path(s.settings, 'alerting') AS alerting
```

Comment (ritbl) [reply managed/services/telemetry/config.default.yml]: sure, good idea

**Review by dliakhov — APPROVED** [review]

**Review by pkadej — APPROVED** [review]

**Pull request merged**